### ISSUE TITLE
coco pretraining hyper-parameters

### ISSUE THREAD

**Issue by zhifanlight** [NVIDIA/retinanet-examples#201]
@yashnv @james-nvidia  Hi, I tried reimplementing ResNet50FPN on COCO2017 with 8x2080Ti. Using the default training command in README.md only got 34.8. 

After checking your pretrained model retinanet_rn50fpn.zip, I found your total iteration is 120k rather than default 90k. However I cannot get 35.8 as reported even if I changed my total iterations to 120k.

I would appreciate it if you could give your full pretrain hyper-parameters, such as lr settings(warmup, gamma, milestones, l2_decay), jitter, color-augmentation, anchor-ious and so on.

**Comment by zhifanlight** [comment]
Setting --iters 120000 --milestones 80000 100000 and keeping other parameters as default, I finally got 35.8.
